NtupleProducer/plugins/HTauTauNtuplizer.cc

```diff
@@ -3827,7 +3832,7 @@ void HTauTauNtuplizer::endJob(){
     for(int i=1;i<=ntauIds;i++){
     hTauIDs->GetXaxis()->SetBinLabel(i,tauIDStrings[i-1].Data());
   }
-
+  std::cout<<"Skipped events due to empty lhe weights: "<<empty_lheweights<<std::endl;
```

Comment:
Print the line only if lheweights is different from zero, to avoid future confusion in most samples.